package.json

```diff
@@ -1,61 +1,48 @@
 {
   "name": "svelte-seo",
-  "version": "1.4.1",
-  "svelte": "src/SvelteSeo.svelte",
-  "module": "dist/index.min.mjs",
-  "main": "dist/index.min.js",
-  "types": "types/index.d.ts",
-  "repository": "git@github.com:artiebits/svelte-seo.git",
+  "version": "0.0.1",
   "license": "MIT",
+  "repository": "git@github.com:artiebits/svelte-seo.git",
   "scripts": {
-    "build": "rollup -c",
-    "dev": "rollup -c rollup.config.dev.js -w",
-    "start": "sirv dev",
-    "cy:run": "cypress run",
-    "cy:open": "cypress open",
-    "test": "run-p --race dev cy:run",
-    "prepublishOnly": "yarn build"
-  },
-  "files": [
-    "dist",
-    "src/SvelteSeo.svelte",
-    "types"
-  ],
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,jsx,json,css,scss,md}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "dependencies": {
-    "schema-dts": "^0.8.2"
+    "dev": "vite dev",
+    "build": "svelte-kit sync && svelte-package",
+    "prepublishOnly": "echo 'Did you mean to publish `./package/`, instead of `./`?' && exit 1",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./jsconfig.json --watch",
+    "test": "playwright test",
+    "lint": "prettier --plugin-search-dir . --check . && eslint .",
+    "format": "prettier --plugin-search-dir . --write .",
+    "preview": "vite build && vite preview"
   },
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^12.0.0",
-    "@rollup/plugin-node-resolve": "^8.0.0",
-    "@tsconfig/svelte": "^1.0.0",
-    "cypress": "^5.0.0",
-    "husky": "^4.2.5",
-    "lint-staged": "^10.2.11",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.0.5",
-    "prettier-plugin-svelte": "^1.1.0",
-    "rollup": "^2.3.4",
-    "rollup-plugin-livereload": "^1.0.0",
-    "rollup-plugin-svelte": "^7.1.0",
-    "rollup-plugin-terser": "^6.1.0",
-    "sirv-cli": "^1.0.6",
-    "svelte": "^3.49.0"
+    "@playwright/test": "^1.29.2",
+    "@sveltejs/adapter-auto": "^1.0.0",
+    "@sveltejs/kit": "^1.0.7",
+    "@sveltejs/package": "1.0.2",
+    "eslint": "^8.31.0",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-svelte3": "^4.0.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.1.0",
+    "prettier": "^2.8.2",
+    "prettier-plugin-svelte": "^2.9.0",
+    "svelte": "^3.55.0",
+    "svelte-check": "^3.0.1",
+    "tslib": "^2.4.1",
+    "typescript": "^4.9.4",
+    "vite": "^4.0.4"
   },
+  "type": "module",
   "keywords": [
     "svelte",
     "sapper",
     "seo",
     "ssr"
-  ]
+  ],
+  "lint-staged": {
+    "*.svelte": "eslint --cache --fix",
+    "*.{css,md,svelte}": "prettier --write",
+    "*.js": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```

Comment:
why like this?

why not?
```
    "*.{js,svelte}": "eslint --cache --fix",
    "*.{js,md,svelte}": "prettier --write"
```

Reply:
Hope you've heard of `npm create svelte@latest`? When you create a default SvelteKit  project, it includes those by default.

Reply:
and why you didn't add ts file?

Reply:
> why some files are js, and some are ts?

There are no ts files in the repo

Reply:
Why would I add a ts file?

Reply:
> Hope you've heard of `npm create svelte@latest`? When you create a default SvelteKit project, it includes those by default.

okay, but there is no css in this repo and I was really curious why you have to do 

"*.svelte": "eslint --cache --fix",
"*.js": "eslint --cache --fix"

insted of 

 "*.{js,svelte}": "eslint --cache --fix"

let's leave it

Reply:
> There are no ts files in the repo

strange, I see 4 ts files in your pull request

Reply:
First of all, I have changed the eslint to reflect your changes.
The only other typescript files are the one generated by SvelteKit

Reply:
even if a file generated by SvelteKit it should be linted. but you can leave it, I will add missing files (ts, json) later myself